hyrisecockpit/api/app/metric/controller.py

```diff
@@ -0,0 +1,88 @@
+"""Controllers for metrics."""
+from flask import request
+from flask_accepts import accepts, responds
+from flask_restx import Namespace, Resource
+
+from .interface import TimeIntervalInterface
+from .schema import (
+    DetailedQueryInformationSchema,
+    LatencySchema,
+    QueueLengthSchema,
+    ThroughputSchema,
+)
+from .service import MetricService
+
+api = Namespace("Metric", description="Metric data.")
+
+
+@api.route("/throughput")
+class ThroughputController(Resource):
+    """Controller of throughput data."""
+
+    @accepts(
+        dict(name="startts", type=int),  # noqa
+        dict(name="endts", type=int),  # noqa
+        dict(name="precision", type=int),  # noqa
+        api=api,
+    )
+    @responds(schema=ThroughputSchema(many=True), api=api)
+    def get(self):
```

Comment:
If it responds with the `ThroughputSchema`, I suggest adding a return type with the `Throughput` model like:
```suggestion
    @responds(schema=ThroughputSchema(many=True), api=api)
    def get(self) -> List[Throughput]:
```